Add profile for Creality Ender 3 S1 Pro and Verbatim PLA

rk0n opened the pull request:
Add Profile for Ender 3 S1 Pro

- [X] Verify bed size is 220mmx220mm (Same as S1)
- [X] Test 'Alpha' thumbnail

rtyr commented:
What is the difference between S1 and S1 Pro?

Do you have that printer? (if tested we don't have to use "alpha" label).

`renamed_from` line should be removed from Verbatim profile
`filament_spool_weight` is not correct (it is supposed to be weight of empty spool).

FYI @pmjdebruijn

pmjdebruijn commented:
I'm guessing the S1 Pro is equipped with the Sprite Pro extruder/hotend, which is essentially an all metal hotend variant. 

@rk0n how did you come to 200C for Verbatim PLA, while very plausible, did you do a temperature tower test?

Also why is the bed temp set to 65C, which is highly implausible if the base temp is 200C...

Please set the color to #001ca8

pmjdebruijn commented:
It seems filament_density isn't correct either, Verbatim's website claims 1.24 like most run of the mill PLAs.

pmjdebruijn commented:
@rk0n thanks for the changes...

However, filament_spool_weight = 500 is unlikely to even remotely correct (somewhere 200-300 is typical), unless you've actually measured an empty spool, please remove it.

I also noticed you've added Verbatim PLA only to your own printer (through default_materials), either don't add it to any, or add it to all printers.

@rtyr, maybe there's a better way to handle this, so default_materials doesn't have to be specifically defined for each printer?

rk0n commented:
> What is the difference between S1 and S1 Pro?
Max. Hot End Temp: 255ºC vs. 300 ºC
Feeder System: Bowden (MK8) vs. Direct all-metal (Sprite)

> Do you have that printer? (if tested we don't have to use "alpha" label).

- [X] change to "beta" label

> `renamed_from` line should be removed from Verbatim profile `filament_spool_weight` is not correct (it is supposed to be weight of empty spool).
>
> FYI @pmjdebruijn

- [X] removed renamed_from
- [X] fix filament_spool_weight

> I'm guessing the S1 Pro is equipped with the Sprite Pro extruder/hotend, which is essentially an all metal hotend variant.
>
> @rk0n how did you come to 200C for Verbatim PLA, while very plausible, did you do a temperature tower test?
>
> Also why is the bed temp set to 65C, which is highly implausible if the base temp is 200C...

@pmjdebruijn now I created a temperature tower test and set the temps properly
- [X] temp tower test
- [X] adjust heatbed temp
- [X] adjust extruder

> Please set the color to #001ca8
- [X] set color to #001ca8

> It seems filament_density isn't correct either, Verbatim's website claims 1.24 like most run of the mill PLAs.

- [X] fix filament density


> However, filament_spool_weight = 500 is unlikely to even remotely correct (somewhere 200-300 is typical), unless you've actually measured an empty spool, please remove it.

- [X] removed filament_spool_weight


> I also noticed you've added Verbatim PLA only to your own printer (through default_materials), either don't add it to any, or add it to all printers.

- [X] add Verbatim PLA to all printers

pmjdebruijn commented:
I only really care it's consistent between all printers.

Given that the number of filament presets isn't so large, it's just convenient to have them all enabled by default. Obviously this won't scale infinitely, but I don't think we're there yet that we have to make choices on that particular topic.

rk0n commented:
@rtyr @pmjdebruijn anything new on this topic?

pmjdebruijn commented:
@rk0n sorry for the wait

@rtyr looks fine to me now... so pull at your convenience...
